plugins/blip/basics/components/CalendarContainer.js

```diff
@@ -74,6 +80,20 @@ var CalendarContainer = React.createClass({
       </div>
     );
   },
+  renderFingerstickLegend: function() {
+    return (
+      <div className='Calendar-legend'>
+        <div className='Calendar-legend-series'>
+          {dotUtil.drawDot(1,true)}
+          <span>Manual</span>
+        </div>
+        <div className='Calendar-legend-series'>
+          {dotUtil.drawDot(1,false)}
+          <span>Linked</span>
+        </div>
+      </div>
+    );
+  },
```

Comment:
Can we factor this out into a separate component that is referenced in `state.js` as part of the component (similar to the plan for the bolus type selector RadioGroup, IIRC)? That's a much more future-proof scheme for legends, as we add more and more components.